changelog.txt

```diff
@@ -1,3 +1,15 @@
+4.1.0
+=============
+New Features:
+- **MSAL.NET now provides options to control the system web browser**. From MSAL.NET 4.0.0, you have been able to use the interactive token acquisition with .NET Core, by delegating the sign-in and consent part to the system web browser on your machine. MSAL.NET 4.1, brings improvements to this experience by helping you run a specific browser if you wish, and by giving you ways to decide what to display to the user in case of a successful authentication, and in case of failure. [More information about this feature here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/msal-net-4.1#improved-experience-with-the-system-web-browser-on-net-core)
+- **MSAL.NET now supports ClientAssertions**. See [issue 1188](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1188)
```

Comment:
nit: inconsistency on [] for issues.  This one it's [issue 1188] and the other one it's [#1148]

Reply:
I think you can just do [1148] and github will auto-link it.

Reply:
thanks...we used to do it like that but when the repos merged, we had to specify the url..will get back to doing it this way again. thx@

---
In reply to: [298357591](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1238#discussion_r298357591) [](ancestors = 298357591)